v2/tools/generator/internal/codegen/pipeline/testdata/TestGolden_InjectConvertibleInterface/microsoft.person-v20200101.golden

```diff
@@ -98,6 +107,9 @@
  	}
  	destination.Status = status
  
+ 	// TypeMeta
+ 	destination.TypeMeta = person.TypeMeta
```

Comment:
I thought you needed to avoid doing this, since it ends up copying the GVK from one version to another?

Reply:
Yeah, I had changes that I hadn't pushed. Now fixed.